kotlin-authentication-action-archetype/src/main/resources/archetype-resources/src/main/kotlin/__pluginName__/__pluginName__AuthenticationAction.kt

```diff
@@ -0,0 +1,23 @@
+package ${package}.${pluginName}
+
+import se.curity.identityserver.sdk.attribute.AuthenticationAttributes
+import se.curity.identityserver.sdk.authentication.AuthenticatedSessions
+import se.curity.identityserver.sdk.authenticationaction.AuthenticationAction
+import se.curity.identityserver.sdk.authenticationaction.AuthenticationActionResult
+import se.curity.identityserver.sdk.service.authenticationaction.AuthenticatorDescriptor
+
+class ${pluginName}AuthenticationAction(private val _configuration: ${pluginName}AuthenticationActionConfig): AuthenticationAction
```

Comment:
Does `pluginName` gets capitalized?

Reply:
No, we currently don't do any transformation on the plugin name, but if you provide a capitalized name for the plugin this will work well. (though you then have to rename the package so it starts with lower case)